http/rest-client-reactive/src/test/java/io/quarkus/ts/http/restclient/reactive/ReactiveRestClientIT.java

```diff
@@ -314,6 +314,30 @@ public void clientReactiveRetry() {
                 .body(containsString("visitor of this page"));
     }
 
+    @Test
+    @Tag("https://github.com/quarkusio/quarkus/issues/43364")
+    public void testUploadFileWithoutMediaType() {
+        String expectedResponse = app.given()
+                .get("/test/upload/without-media-type")
+                .then()
+                .statusCode(200)
+                .extract().asString();
+
+        assertEquals("File received: fileTest.txt content: This is sample test content", expectedResponse);
```

Comment:
Can't you just check like this `body(containsString("String"))` without extracting the response.